docs/Build.md

```diff
@@ -2,40 +2,95 @@
 
-## Raspberry PI
-
-We mainly use a Raspberry PI as the computing device behind the leek-box.
-
-### Wifi
-
-To simplify the setup of the PIs wifi we recommend the [comitup](https://github.com/davesteele/comitup) library by davesteele.
-
-#### Installation
-
-1. Install `comitup` from the PPA as described by [this](https://davesteele.github.io/comitup/ppa.html) guide.
-
-1. Then setup `comitup` like done by [this](https://github.com/davesteele/comitup/wiki/Installing-Comitup) guide.
-
-1. Edit the `/etc/comitup.conf` file and set the following values:
-    ```conf
-    ap_name=leek-<nnn>
-    ```
-
-1. Finally reboot the PI.
-
-### Install docker
-
-TBD
-
-#### NFC-Reader container setup
-
-This container is used as an interface between the USB-NFC-Reader and the backend.
-
-TBD
-
-#### Backend container setup
-
-This container is the actual backend which is used by the leeks app.
-It has an websocket and REST interface to manage the database which holds all data of the app.
-
-TBD
\ No newline at end of file
+## What do you need?
+* Raspberry PI
+* SD-Card
+* HDMI-Cable
+* Power-Cable
+* Lan-Cable
+* NFC-Reader
+
+## 1. Setting up the Raspberry PI
+Since the Raspberry is ideally hidden in a fancy box, it is recommended to install the Raspberry Pi OS Lite. This saves resources, as no graphical user interface needs to be rendered if no monitor is connected most of the time anyway.
+
+### 1.1 Making the SD card bootable
+* Download the Raspberry Pi Imager https://www.raspberrypi.org/software/
+* choose os -> Raspberry Pi OS (other) -> Raspberry Pi OS **Lite**
+* choose SD Card and start writing
+
+After the Raspberry Pi Imager is ready, the SD card can be plugged into the back of the PI. It now needs to be connected to power, LAN, a monitor and a keyboard.
+
+### 1.2 Connecting the PI
+When the Raspberry PI is now fully connected, you should be able to log in for the first time. The default login data  is as follows: </br>
+  ```json
+  user: pi
+  password: raspberry
+  ```
+⚠️ The default keyboard setting is English.
+Europeans must therefore watch their z.
+
+Connect the Pi to the WLAN:
+For the next steps, it is easiest if we connect to the PI via a computer in order to be able to work through the following instructions as quickly as possible. Therefore, we recommend connecting the PI via LAN and establishing an SSH connection.
+
+Activate SSH: <br>
+```linux
+  sudo systemctl enable ssh
+  sudo systemctl start ssh
+```
+Connecting to the Pi <br>
+To obtain the PI's ip address run
+```
+ifconfig
+```
+and connect with a ssh-client like [vsCode](https://www.hanselman.com/blog/visual-studio-code-remote-development-over-ssh-to-a-raspberry-pi-is-butter)
+
+Now that we can comfortably copy and paste commands onto the raspberry, let's get down to the actual wifi setup. <br>
+1. Add the comitup repo as instructed [here](https://davesteele.github.io/comitup/ppa.html)
+2. prepare the pi for the service with
```

Comment:
```suggestion
1. prepare the pi for the service with
```

Reply:
why?

Reply:
If you always use `1.` with markdown, it will automatically count up for you.